site/features.md

```diff
@@ -55,6 +56,28 @@ now re-use the same tools and techniques when deploying applications
 as containers as we would have done when deploying them 'on metal' in
 our data centre.
 
+### <a name="automatic-overlay-selection"></a>Automatic overlay selection
+
+Weave supports two overlay technologies - a custom UDP encapsulation
+('sleeve') featuring optional encryption and underlay PMTU discovery,
+and a VXLAN transport ('fastdp') that leverages the Open vSwitch
+kernel module to provide near-native performance. When connecting to a
+new peer, the router automatically prefers the fastdp overlay under
+the following conditions:
+
+* UDP port 6784 must be reachable (in addition to TCP/UDP 6783)
```

Comment:
We need to mention the 6784 in the README (where we mention 6783). Needs a separate PR on a readme-v1.2 (or whatever naming scheme we used previously).